yarn.lock

```diff
@@ -5,18 +5,22 @@
 "7zip-bin-linux@~1.3.1":
   version "1.3.1"
   resolved "https://registry.yarnpkg.com/7zip-bin-linux/-/7zip-bin-linux-1.3.1.tgz#4856db1ab1bf5b6ee8444f93f5a8ad71446d00d5"
+  integrity sha512-Wv1uEEeHbTiS1+ycpwUxYNuIcyohU6Y6vEqY3NquBkeqy0YhVdsNUGsj0XKSRciHR6LoJSEUuqYUexmws3zH7Q==
```

Comment:
This diff is surprisingly large - I'd guess it's because you're not based on the most recent commits in the development branch.

Reply:
At the time, as far as I remember, I used the head of development branch. But it should be easylaly rebased with the head.